mise.toml

```diff
@@ -10,6 +10,7 @@ yq = "4.44.5"
 shellcheck = "0.10.0"
 direnv = "2.35.0"
 just = "1.37.0"
+node = "20.0.0"
```

Comment:
Any reason not to use the latest LTS? (`22.14.0`)?

Reply:
CI fails with anything other that 20.0.0, with a not very helpful error message:

```
Error: 
   0: failed to install core:rust@1.83.0
   1: [~/project/mise.toml] npm:doctoc@2.2.0: No such file or directory (os error 2)
   2: error resolving versions
```

At this point I'm tempted to just document that people should install `doctoc` manually, instead of introducing such a cursed dependency on CI.